src/components/MultipleFileInputField/MultipleFileInputField.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form, FormControl, IconButton,
+} from '@edx/paragon';
+import classNames from 'classnames';
+import * as FontAwesome from '@fortawesome/free-solid-svg-icons/faTimes';
+import { getSizeInBytes, formatBytes } from './utils';
+
+const MultipleFileInputField = ({
+  input,
+  label,
+  type,
+  description,
+  disabled,
+  required,
+  className,
+  id,
+  meta: { touched, error },
+  ...props
+}) => {
+  const hasError = !!(touched && error);
+  const [size, setSize] = useState('0');
+  const [filesSizeError, setFilesSizeError] = useState(null);
+  const { name: inputName, onChange: inputOnChange, value: inputValues } = input;
+
+  const handleInputChange = (e) => {
+    const file = e.target.files[0];
+    const sizeOfPreviousFiles = getSizeInBytes(size);
+    const totalSize = sizeOfPreviousFiles + file.size;
+    if (totalSize > 2097152) {
```

Comment:
Can we use a constant here instead of a hard-coded value? You can either define it in the same component or in the `constants` file.

```javascript
const MAX_FILE_SIZE = 2097152;


if (totalSize > MAX_FILE_SIZE)

Reply:
done